public/js/load_comment.js

```diff
@@ -0,0 +1,104 @@
+
+//const & var
+const addComment = (data)=>{
+  // obj = JSON.parse(data);
+  const comment = document.createElement('div');
+  comment.innerHTML= data;
+  comment.setAttribute("class","comment");
+  document.querySelector("body").appendChild(comment);
+  offset +=2; 
+  console.log("json");
+}
+
+var limit = 2;
+var offset = 0;
+
+//management
+  document
+.querySelector('#load_more')
+.addEventListener("click",(e)=> {
```

Comment:
Codacy has a fix for the issue: [Missing space before =>.](https://app.codacy.com/gh/soutenance14/ST/pullRequest?prid=8161867)
```suggestion
.addEventListener("click",(e) => {
```